src/user.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @fileoverview Genotet user.
+ */
+
+'use strict';
+
+/** @const */
+genotet.user = {};
+
+/**
+ * @typedef {{
+ *   username: (string|undefined),
+ *   password: string,
+ *   sessionId: (string|undefined),
+ *   expiration: (string|undefined)
+ * }}
+ */
+genotet.Cookie;
+
+/**
+ * Initializes the user auth.
+ */
+genotet.user.init = function() {
+  if (!document.cookie ||
```

Comment:
You may want to test for a specific cookie (or a key-value) rather than whether the cookie exists? Theoretically there could be other stuffs on the page writing cookies as well.